q2cli/commands.py

```diff
@@ -8,13 +8,13 @@
 
 import click
 
+from q2cli.click.command import BaseCommandMixin
+from q2cli.core.config import CONFIG
+
```

Comment:
Please revert this, we wound up not needing to make any changes to the import order, so maybe its best to let sleeping dragons lie - thanks!